core/sdk/src/api/manifest/device_manifest.rs

```diff
@@ -50,32 +50,28 @@ pub struct RippleConfiguration {
     pub distribution_platform: String,
     pub distribution_id_salt: Option<IdSalt>,
     pub form_factor: String,
-    #[serde(default = "default_values_default")]
+    #[serde(default)]
```

Comment:
I switched DeviceManifest and all its sub-structs to be `Default` capable. So that a unit test can just create a default manifest.
Instead of just doing defaults in serde, now serde just does the default in code.